### ISSUE TITLE
`jd` - JSON diff and patch 

### ISSUE THREAD

**Issue by blaggacao** [NixOS/nixpkgs#140311]
**Project description**
jd is a commandline utility and Go library for diffing and patching JSON values.

**Metadata**

* homepage URL: https://github.com/josephburnett/jd
* source URL: https://github.com/josephburnett/jd
* license: mit
* platforms: unix, linux, darwin

**Comment by roberth** [comment]
Did you try to package it? Golang is usually straightforward to package.

**Comment by blaggacao** [comment]
@roberth No, not yet. I'm in a hell of a multi-flank wrestle against different simultaneous tasks, atm. :smile:

But you are right, it should be straight forward.

I would have hoped to somehow crowd-source this item, though, cause I can't take another context switch atm. :smile:

**Comment by bryanasdev000** [comment]
Sounds like a nice tool, maybe I can take a look.

**Comment by bryanasdev000** [comment]
@blaggacao done at https://github.com/NixOS/nixpkgs/pull/140687 , could you give me an input at the package name and if you want to maintain it too?

We already have a different `jd` package (https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/development/tools/jd/default.nix#L17).

Also, I'm not including the demo web UI, not sure if it's needed.

**Comment by blaggacao** [comment]
@bryanasdev000 Thanks a lot! :rocket: I really appreciate it. :pray:
